README.md

```diff
@@ -1,2 +1,70 @@
 # purescript-elmish-enzyme
-A testing library for Elmish which uses Enzyme
+
+`elmish-enzyme` is a testing library for Elmish which provides bindings for [Enzyme](https://enzymejs.github.io/enzyme/), a React testing library. Enzyme allows mounting a component, then interacting with it and testing properties of the component. This library offers a monadic API around Enzyme, which lets you write tests like this:
+
+```purs
+import Elmish.Enzyme (testComponent, text, find, at, simulate, (>>))
+
+someTest = it "has two buttons with the right text" do
+  testComponent myComponent do
+    find ".card" >> at 3 >> find "button" >> simulate "click"
+    find ".modal" >> text >>= shouldEqual "Modal content"
+```
+
+## Getting Started
+
+To start using this library, you’ll need to install Enzyme, and [JSDOM](https://github.com/jsdom/jsdom).
+
+```
+npm install enzyme jsdom global-jsdom --save-dev
```

Comment:
You also need to install `enzyme` itself. I think it might be better to combine all of the `npm install` commands into one and then add a bulleted list describing what each dependency is for.

Reply:
Enzyme is included here, but I didn’t want someone to copy and paste a specific adapter without thinking about which one they need. I could make it clear by doing something like this, maybe:

```
npm install enzyme jsdom global-jsdom <REACT_VERSION_ADAPTER> --save-dev
```

Reply:
Oh, indeed, I totally missed `enzyme` for some reason 🤦